### PR TITLE
implement sortMultisigActions

### PR DESCRIPTION
**Pull request by Rekard0**
## Description

- add wallet always at the top

- if removed wallet count goes below current (old or active on contract) minApproval, then first modify (i.e lower) minApproval then remove address

Task: [APP-1737](https://aragonassociation.atlassian.net/browse/APP-1737)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.
- [x] I have tested my code on the test network.

[APP-1737]: https://aragonassociation.atlassian.net/browse/APP-1737?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/web-app/src/utils/library.ts [hunk 1]
```diff
@@ -255,6 +256,7 @@ export async function decodeRemoveMembersToAction(
 
   return Promise.resolve({
     name: 'remove_address',
+    previousLength: 0,
```
Comment (Fabricevladimir) [inline packages/web-app/src/utils/library.ts]: Are we still using this? I think we're getting it from the `UseDaoMembers` when the sort function is called.

Comment (Rekard0) [reply packages/web-app/src/utils/library.ts]: nice catch, amended

**Review by Fabricevladimir — APPROVED** [review]
LGTM 🚀

**Review by sepehr2github — APPROVED** [review]
Thanks, Nice job

**Comment by Fabricevladimir** [comment]
This will come after the initial release after we've determined the proper UX

**Pull request closed without merging**